cabal.project

```diff
@@ -34,8 +34,8 @@ packages:
 
 extra-packages: Cabal
 
-program-options
-  ghc-options: -Werror
+-- program-options
+--   ghc-options: -Werror
```

Comment:
I think this should remain enabled, but it doesn't matter for the release as the `cabal.project` is not distributed.